src/module.c

```diff
@@ -9585,7 +9579,7 @@ int moduleLoad(const char *path, void **module_argv, int module_argc) {
     int (*onload)(void *, void **, int);
     void *handle;
     RedisModuleCtx ctx;
-    moduleCreateContext(&ctx, NULL); /* We pass NULL since we don't have a module yet. */
+    moduleCreateContext(&ctx, NULL, REDISMODULE_CTX_NONE); /* We pass NULL since we don't have a module yet. */
```

Comment:
maybe preserve that comment in the other occurrence of this case (to be on the safe side)

Reply:
the only other occurrence is RM_GetThreadSafeContext and even then sometimes we have a module and sometimes we don't.. i don't see a value of having the comment there too